Contemos2018

mauricioalfarodev opened the pull request:
corrige error validacion

mxgxw committed:
Agregado codigo para identificar centroide de codigo de barra

mxgxw committed:
Agregado instalacion base de CakePHP 3

mxgxw committed:
Migradoocodigo para generacion de tokens y cambios en web.

mxgxw committed:
Migrado seleccionador de imagenes.

mxgxw committed:
Migracion de validador a CakePHP3 Completada.

mxgxw committed:
Agregado SQL para version funcional.

klhernandez42 committed:
cambiando redaccion y resolviendo reporte de texto perdido en el anuncio de confirmacion de acta enviada.

mxgxw committed:
Merge pull request #11 from karlalhdz/contemos2018

mejorando redaccion y resolviendo reporte de texto perdido.

salvadoresc committed:
Update default.ctp

salvadoresc committed:
agregado segundo boton

agregado segundo boton

salvadoresc committed:
ancho contenedor imagenes actas

las imagenes son mas grandes asi que se incremento el espacio disponible

mxgxw committed:
Modificado para utilizar pedacitos png

mxgxw committed:
Merge remote-tracking branch 'origin' into contemos2018

mxgxw committed:
Merge branch 'contemos2018' of https://github.com/mxgxw/crowdvalidation into contemos2018

salvadoresc committed:
boton ilegible

salvadoresc committed:
cambio de tamano para imagen

salvadoresc committed:
ancho de imagen modificado

salvadoresc committed:
Merge branch 'contemos2018' of https://github.com/mxgxw/crowdvalidation into contemos2018

salvadoresc committed:
boton ilegible

salvadoresc committed:
correccion imagen banner y fondo

salvadoresc committed:
fondo pagina

salvadoresc committed:
correccion input excediendo ancho en moviles

mxgxw committed:
Agregado SQL para consultas de reportes

mxgxw committed:
Habilitado reporte de datos ilegibles.

klhernandez42 committed:
Cambiando redaccion para comprender casillas vacias o con guiones.

mxgxw committed:
Merge pull request #12 from karlalhdz/contemos2018

Cambiando redacción para comprender casillas vacias o con guiones.

mxgxw committed:
Agregados resultados de prueba.

mxgxw committed:
Agregado SQL de pruebas

mxgxw committed:
Agregado DUMP de resultado de digitaciones

mxgxw committed:
Corregido error de ortografía en README.md

mxgxw committed:
Agregados archivos de configuracion

mxgxw committed:
Merge branch 'contemos2018' of https://github.com/mxgxw/crowdvalidation into contemos2018

kadejo committed:
Update manifiesto.ctp

mxgxw committed:
Codigo viejo eliminado. Ya se encontraba en el archivo.

salvadoresc committed:
input number

se cambio el input a number, estaba en texto

mxgxw committed:
Eliminado .htaccess de webroot ya que requiere cambios locales

mxgxw committed:
Corregido .gitignore para no incluir .htaccess locales

mxgxw committed:
Merge branch 'contemos2018' of https://github.com/mxgxw/crowdvalidation into contemos2018

mxgxw committed:
Merge pull request #13 from kadejo/patch-1

Correcciones al manifiesto.

mxgxw committed:
Agregados archivos de base CakePHP

mxgxw committed:
Modificado .gitignore y correccion en Layout

salvadoresc committed:
ganalitycs

mxgxw committed:
Agregado SQL de reset. Corregidos codigos de error.

manasv committed:
Bootstrap Updated

salvadoresc committed:
Merge branch 'contemos2018' into contemos2018

salvadoresc committed:
Merge pull request #14 from manasv/contemos2018

Bootstrap Updated

salvadoresc committed:
referencia css

salvadoresc committed:
imagen fondo

salvadoresc committed:
fix bg

salvadoresc committed:
fix estilos eliminados

salvadoresc committed:
revert css

salvadoresc committed:
version jquery

salvadoresc committed:
faq page

salvadoresc committed:
banner via cake

salvadoresc committed:
color container para paginas

salvadoresc committed:
varios

clase a navegacion de inicio, faq y manifiesto, eliminar jumbotron

salvadoresc committed:
correccion de columnas

salvadoresc committed:
remover doble input

salvadoresc committed:
doble input fix

salvadoresc committed:
revertir doble input

salvadoresc committed:
double input remove

salvadoresc committed:
test styles

salvadoresc committed:
clases center

salvadoresc committed:
centrado card

salvadoresc committed:
reduccion logo

salvadoresc committed:
ediciones varias

boton de acta irregular, correcion de menus y footer pagina agradecimientos

salvadoresc committed:
fix navbar

salvadoresc committed:
load new if errror

salvadoresc committed:
test bootstrap version

salvadoresc committed:
eliminado button togler

salvadoresc committed:
padding logo

salvadoresc committed:
fix navigation

salvadoresc committed:
fix logo

salvadoresc committed:
fix div navigation

salvadoresc committed:
cambios manasv

salvadoresc committed:
fix input

salvadoresc committed:
fix input

salvadoresc committed:
contenido faq

salvadoresc committed:
prueba link

salvadoresc committed:
link manifiesto

salvadoresc committed:
full navigation

salvadoresc committed:
clase navegacion manifiesto fix

salvadoresc committed:
correcciones antes de push

salvadoresc committed:
restriccion numeros negativos

salvadoresc committed:
max 999

salvadoresc committed:
comment bootstrapjs

salvadoresc committed:
ancho input

mxgxw committed:
Agregado FAQ y Manifiesto

salvadoresc committed:
fix validacion input

salvadoresc committed:
Merge branch 'contemos2018' of https://github.com/mxgxw/crowdvalidation into contemos2018

mxgxw commented:
¡Hola! Vamos a dejarlo en HOLD hasta el siguiente release, al menos que quede un poco más refinado para integrarlo al master :)

mxgxw committed:
Corregidas URL de SQL Dumps.

mxgxw committed:
Merge branch 'contemos2018' of https://github.com/mxgxw/crowdvalidation into contemos2018

mauricioalfarodev committed:
Update home.ctp

corrige validacion

mxgxw committed:
Merge pull request #19 from mauri550/contemos2018

Corrije return en home.ctp

mxgxw committed:
Agregado HASH para proteger direcciones IP

mxgxw committed:
Merge branch 'contemos2018' of https://github.com/mxgxw/crowdvalidation into contemos2018

guillegr123 committed:
Adicion de script SQL para crear y poblar tablas de deptos., partidos y diputados.

Las tablas de departamentos y partidos se cargan en base a la tabla inicial de diputados, la cual se genero a partir del documento de Excel de diputados.

El script puede correrse las veces que se quiera, re-creando las tablas en cada ocasion, mientras estas tablas no tengan relacion con otras tablas externas al script.

guillegr123 committed:
Cambio de campos en tabla diputados a nulables.

mxgxw committed:
Merge pull request #20 from guillegr123/contemos2018

Adicion de script SQL para crear y poblar tablas de deptos., partidos…

kadejo committed:
Nuevos ajustes (identificados por belcboo y otros)

mxgxw committed:
Merge pull request #22 from kadejo/patch-3

Nuevos ajustes (identificados por belcboo y otros)

kadejo committed:
update por incompletitud de actas en origen TSE

manasv committed:
Fix centrado de botones e imagen

manasv committed:
Testing Autofocus

mxgxw committed:
Merge pull request #23 from kadejo/patch-4

update por incompletitud de actas en origen TSE

mxgxw committed:
Merge pull request #25 from manasv/contemos2018

Testing Autofocus

manasv committed:
Removed Shadows, Modified Background

manasv committed:
Fix Footer

salvadoresc committed:
eliminacion boton acta irregular

salvadoresc committed:
reduccion texto link

salvadoresc committed:
Merge pull request #26 from manasv/contemos2018

Removed Shadows, Modified Background

mxgxw committed:
Agregado fix para lentitud en busqueda aleatoria

kadejo committed:
bugcitos y mejoras para la digestión mediática

mxgxw committed:
Merge pull request #27 from kadejo/patch-5

bugcitos y mejoras para la digestión mediática

salvadoresc committed:
agradecmientos

salvadoresc committed:
Merge branch 'contemos2018' of https://github.com/mxgxw/crowdvalidation into contemos2018

salvadoresc committed:
fix css

salvadoresc committed:
ejemplo perfiles

salvadoresc committed:
float en profile

salvadoresc committed:
margin profile

salvadoresc committed:
fix titulo profile

mxgxw committed:
Agregado controlador API para consultas

mxgxw committed:
Agregada consulta por voto preferencial

salvadoresc committed:
ocultar agradecimientos temporalmente

mxgxw committed:
Agregada extraccion de totales de marcas en API

mxgxw committed:
Corregido bug en controlador API

salvadoresc committed:
fix profile

kadejo committed:
cambio canal de slack por correo info para apoyar

mxgxw committed:
Merge pull request #28 from kadejo/patch-6

cambio canal de slack por correo info para apoyar

salvadoresc committed:
datatable test

salvadoresc committed:
Merge branch 'contemos2018' of https://github.com/mxgxw/crowdvalidation into contemos2018

salvadoresc committed:
fotos agradecimientos

salvadoresc committed:
datatable

salvadoresc committed:
prueba dataset

salvadoresc committed:
datatable prueba

salvadoresc committed:
datatable test

mxgxw committed:
Cambios a formato de datos de salida para funcionamiento con DataTables

mxgxw committed:
Agregado cambio de formato

mxgxw committed:
Agregado cambio de formato

mxgxw committed:
Correccion de codigo en API

mxgxw committed:
Agregado cambio map a array_map

mxgxw committed:
Llamada a arreglo antes de convertir

mxgxw committed:
Agregado llamada a arreglo

mxgxw committed:
Agregada referencia para uso de funcion dentro de la instancia.

mxgxw committed:
Corregido nombre de objeto en clase de API

mxgxw committed:
Prueba de formato JSON encoding

mxgxw committed:
Agregado formato datos

mxgxw committed:
Correccion para re-mapear resultados de la consulta

mxgxw committed:
Agregado formato directo en el arreglo de salida

salvadoresc committed:
mejorando datatables

salvadoresc committed:
datatable espanol

salvadoresc committed:
datatable

salvadoresc committed:
datatable

salvadoresc committed:
datatable

salvadoresc committed:
datatable

salvadoresc committed:
fontsize datatable

salvadoresc committed:
th color datatable

salvadoresc committed:
borde header y footer datatable

salvadoresc committed:
borde datatable

salvadoresc committed:
reporte diputados y clase activa de datatable

salvadoresc committed:
menu datatable

salvadoresc committed:
texto navegacion cambiado

mauricioalfarodev committed:
Update datatable.ctp

Tabla responsive

mauricioalfarodev committed:
Update datatable.ctp

responsive a tabla con bootstrap

mauricioalfarodev committed:
CSS responsive + class

mauricioalfarodev committed:
rollback

mauricioalfarodev committed:
rollbak con tabla

mauricioalfarodev committed:
probando chartjs

mauricioalfarodev committed:
C

mauricioalfarodev committed:
otra

mauricioalfarodev committed:
primerasPruebas

mauricioalfarodev committed:
cierre

mauricioalfarodev committed:
subiendo fotos, cambio funcion js y cambio css

mauricioalfarodev committed:
pathname

mauricioalfarodev committed:
url sitio

mauricioalfarodev committed:
url diputados

mauricioalfarodev committed:
sin cambiar tamaño img

mauricioalfarodev committed:
ubicacion y nuevo tamaño de imagenes

mauricioalfarodev committed:
padding +3 en linea

mauricioalfarodev committed:
padding

mauricioalfarodev committed:
otro padding

mauricioalfarodev committed:
alinear al centro

mauricioalfarodev committed:
colores

mauricioalfarodev committed:
cambiar degradado

mauricioalfarodev committed:
otro dip para prueba

mauricioalfarodev committed:
colores mas limpios

mauricioalfarodev committed:
color3 erro funcion crear degradado

mauricioalfarodev committed:
color header

mauricioalfarodev committed:
textos

mauricioalfarodev committed:
.col-xs-6 .col-sm-4

mauricioalfarodev committed:
.col-xs-6 .col-sm-4

mauricioalfarodev committed:
css

mauricioalfarodev committed:
reparar 3 filas css

mauricioalfarodev committed:
css

mauricioalfarodev committed:
tamaño chart

mauricioalfarodev committed:
Cambio version js bootstrap

mauricioalfarodev committed:
cambio tamaño chart

mauricioalfarodev committed:
CAMBIOS CHART

mauricioalfarodev committed:
ultima mala practica

mauricioalfarodev committed:
se mira vien todo menos textos

kadejo committed:
una falta de ortografía apenas

mxgxw committed:
Merge pull request #34 from kadejo/patch-7

una falta de ortografía apenas

mauricioalfarodev committed:
js

mauricioalfarodev committed:
Merge branch 'contemos2018' of https://github.com/mxgxw/crowdvalidation

mxgxw committed:
Removidas columnas no informativas. Agregadas consultas SQL.
